Localize/lang/strings.json

```
@@ -782,6 +780,7 @@
   "_+JCU+Y.comment": "Tooltip for on-prem gateway connection checkbox",
   "_+W2vTp.comment": "Placeholder description for a newly inserted Email parameter",
   "_+W9fkq.comment": "Timezone value ",
```

Should trailing whitespace exist here?

```suggestion
  "_+W9fkq.comment": "Timezone value",
```

---

yeah I think so